### ISSUE TITLE
setOnColor() shows a different color instead

### ISSUE THREAD

**Issue by javivdev** [BeppiMenozzi/TriStateToggleButton#3]
When I assign the OFF, MID and ON colors on the xml file the colors are displayed correctly, but when I set them programmatically through setOnColor(), setMidColor() and setOffColor(), the color displayed is another.

The way I set the color is:
`tstbAppState.setOnColor(R.color.yellow);`

By the way, thanks for the library...

**Comment by Swavis** [comment]
Yes, same issue, when i am trying to set transparent color (#26000000) through xml, it shows **Black** color instead.
@jsmeke did you find the solution..?
